src/main/java/net/porillo/GlobalWarming.java

```diff
@@ -133,7 +137,13 @@ public void onDisable() {
     private void registerCommands() {
         commandManager.enableUnstableAPI("help");
         commandManager.setFormat(MessageType.HELP, ChatColor.GREEN, ChatColor.DARK_GREEN, ChatColor.AQUA);
-        commandManager.getCommandContexts().registerIssuerOnlyContext(GPlayer.class, c -> tableManager.getPlayerTable().getPlayers().get(c.getPlayer().getUniqueId()));
+        commandManager.getCommandContexts().registerIssuerOnlyContext(GPlayer.class, c -> {
+            CommandSender sender = c.getSender();
+            if (sender instanceof Player) {
+                return tableManager.getPlayerTable().getPlayers().get(c.getPlayer().getUniqueId());
```

Comment:
This is a nice solution, perhaps we could create a generic GPlayer to be used for any CommandSender?

Would just need to be careful for when we actually use the GPlayer in the command classes, maybe just add an additional check for all the commands which strictly need an ingame player.

Reply:
We can replace GPlayer with CommandSender wherever we want everybody to execute the command and ACF will resolve it for us

Reply:
We might have to change isCommandAllowed method, we could replace that method with a custom ACF flag, making it cleaner and simpler